### PR TITLE
Bug report for collection.findByIds$(...)

### PR DESCRIPTION
**Pull request by Hideman85**
Bug report submission for https://github.com/pubkey/rxdb/issues/3659

### CODE REVIEW

**Comment by Hideman85** [comment]
I think missed something I did commit a new file `test/unit/bug-findbyids.test.ts` to not override the sample file but it seems to be ignored by testing suite (except coding style of course :D)

**Comment by Hideman85** [comment]
Now the test is failing according to what I observed

File: test/unit/bug-findbyids.test.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+describe('bug-report.test.js', () => {
+    it('should fail because it reproduces the bug', async () => {
+
+        /**
+         * If your test should only run in nodejs or only run in the browser,
+         * you should comment in the return operator and addapt the if statement.
+         */
+        if (
+            config.platform.isNode() // runs only in node
```
Comment (pubkey) [inline test/unit/bug-findbyids.test.ts:36]: Does it also fail in node?

Comment (Hideman85) [reply test/unit/bug-findbyids.test.ts:36]: I don't know I was not able to run node locally but yes this line is useless, I should remove it

Comment (Hideman85) [reply test/unit/bug-findbyids.test.ts]: The result on node is exactly the issue I'm facing too.
The observer get called for each single update and the map do not have the docs.
```
Got update Map(4) {
  'a' => undefined,
  'b' => undefined,
  'c' => undefined,
  'd' => undefined
}
```
Then this is a bug with `findByIds$`

**Pull request closed without merging**